ta/ta.mk

```diff
@@ -129,6 +116,12 @@ $(conf-mk-file-export): $(conf-mk-file)
 	$(q)echo CFG_ARM32_$(sm-$(@)) := $(CFG_ARM32_$(sm-$(@))) >> $@
 	$(q)echo CFG_ARM64_$(sm-$(@)) := $(CFG_ARM64_$(sm-$(@))) >> $@
 	$(q)echo CFG_TA_FLOAT_SUPPORT := $(CFG_TA_FLOAT_SUPPORT) >> $@
+	$(q)echo ta_arm32-platform-cppflags := $(ta_arm32-platform-cppflags) >> $@
```

Comment:
This is arm-specific, and we would miss some configurations for non-arm platforms.